clients/drcachesim/simulator/caching_device.cpp

```diff
@@ -112,9 +112,9 @@ caching_device_t::request(const memref_t &memref_in)
         // Make sure last_tag is properly in sync.
         assert(tag != TAG_INVALID &&
                tag == get_caching_device_block(last_block_idx, last_way).tag);
-        stats->access(memref_in, true /*hit*/);
+        stats->access(memref_in, true /*hit*/, TAG_INVALID);
```

Comment:
Maybe we should mention the interface change in the changelog (api/docs/release.dox) since w/o the default value it is now a compatibility change to anyone extending this code.

Reply:
Done.